java/src/server/commands/moves/RollNumberCommand.java

```diff
@@ -9,21 +13,32 @@
  * @author Joel Bradley
  */
 public class RollNumberCommand implements ICommand {
+    private int gameId;
+    private int value;
+    private int playerIndex;
+    private IFacade facade;
 
     /**
      * Constructor
      */
-    public RollNumberCommand() {
-
+    public RollNumberCommand(IFacade facade, RollNumberDTO roll) {
+        this.facade = facade;
```

Comment:
Kyle, Danny and Joel decided that the commands will just have the DTO object, and then in the server you'll break it up